docs/docs/features/endpoints.md

```diff
@@ -12,7 +12,8 @@ OAuth2 Proxy responds directly to the following endpoints. All other endpoints w
 - /oauth2/start - a URL that will redirect to start the OAuth cycle
 - /oauth2/callback - the URL used at the end of the OAuth cycle. The oauth app will be configured with this as the callback url.
 - /oauth2/userinfo - the URL is used to return user's email from the session in JSON format.
-- /oauth2/auth - only returns a 202 Accepted response or a 401 Unauthorized response; for use with the [Nginx `auth_request` directive](../configuration/overview.md#configuring-for-use-with-the-nginx-auth_request-directive)
+- /oauth2/auth - only returns a 202 Accepted response or a 401 Unauthorized response; supports [Nginx `auth_request` directive](../configuration/overview.md#configuring-for-use-with-the-nginx-auth_request-directive) and [Traefik (v2) `ForwardAuth` middleware](../configuration/overview.md#configuring-for-use-with-the-traefik-v2-forwardauth-middleware)
+  - Query String `redirect_signin` - accepts a boolean value to trigger OAuth process on unauthenticated sessions, useful to support reverse proxies that do not support scripting/dynamic variable injection (such as Traefik)
```

Comment:
I'm not sure if we want to document all the querystring available on each endpoint in this page (in order to avoid bloat).

This wouldn't even be the first supported querystring param on this endpoint (it has `allowed_groups` as well). Other endpoints don't document their various querystrings or headers on this page (but I do recall I made an issue to figure out a spot for us to document all supported querystrings and headers for all endpoints in a good place).

Reply:
removed the additional query string parameter doc